en/mkdocs.yml

```diff
@@ -350,6 +350,7 @@ nav:
                 - Git Integration: deploy-and-publish/deploy-on-gateway/choreo-connect/git-integration.md
                 - Configure Multiple Gateways with Choreo Connect: deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md
                 - File Upload via Multipart Requests: deploy-and-publish/deploy-on-gateway/choreo-connect/file-upload-via-multipart-requests.md
+                - HTTP 2.0: deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.md
                 - Token Caching: deploy-and-publish/deploy-on-gateway/choreo-connect/configure-caching.md
                 - Production Deployment Guideline: deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md
                 - Troubleshooting:
```

Comment:
What is this plugin in the below?

[en/src/mkpdfs-mkdocs-plugin](https://github.com/wso2/docs-apim/pull/5958/files#diff-5d1ce82990fe6e68ad54951728764b80518ce7a90f89f9dcf9c8908ad72fe678)

Reply:
removed the unwanted file